src/logic/operate.test.js

```diff
@@ -0,0 +1,8 @@
+import operate from './operate';
+
+describe('operate', () => {
+  it('should return the sum of two numbers when passed "+" as the operation', () => {
+    const result = operate('5', '3', '+');
+    expect(result).toEqual('8');
+  });
+});
```

Comment:
According to the requirements, tests that cover all scenarios of the operate js should be written and I recommend writing more tests to see if other functions are working as expected.